packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1219,6 +1236,15 @@ abstract class PlatformViewController {
   ///  * [PlatformViewsRegistry], which is a helper for managing platform view IDs.
   int get viewId;
 
+  /// True if [create] has not been successfully called the platform view.
+  ///
+  /// This can indicate either that [create] was never called, or that [create]
+  /// was deferred for implementation-specific reasons.
+  ///
+  /// A `false` return value does not necessarily indicate that the [Future]
+  /// returned by [create] has completed, only that creation has been started.
+  bool get awaitingCreation => false;
```

Comment:
I was trying to make this as unlikely as possible to break anything; having a default here means that existing subclasses will not break.

For the current use case, `false` as a default means that subclasses have to opt into the deferred load behavior, which seems reasonable. I can see that being potentially confusing later though if we add other uses. How about this for now with a follow-up shortly after to remove the default, so that we can cherry-pick this non-breaking version, then do the more-likely-to-break-someone version as a master-only change?